packages/design-system/src/themes/default/index.css

```diff
@@ -22,7 +23,7 @@
   --ads-border-radius: 0px;
   /* new variables end here */
 
-  --ads-primary-color: var(--ads-color-orange-500);
+  --ads-primary-color: var(--ads-color-brand);
```

Comment:
If we're replacing all `ads-primary-color` with `ads-color-brand` then why does `ads-primary-color` still exist here? What is the purpose of it, and when should we use it?